jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -1,23 +1,27 @@
 #!/usr/bin/python
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *
+from past.builtins import basestring
+from io import BytesIO
 import binascii
 import copy
 import re
-import sys
+import os
 from jmbitcoin.secp256k1_main import *
 from jmbitcoin.bech32 import *
-from _functools import reduce
-
-is_python2 = sys.version_info.major == 2
 
 ### Hex to bin converter and vice versa for objects
 def json_is_base(obj, base):
```

Comment:
So with your changes to serialize/deserialize, I believe you no longer use json_is_base nor json_changebase so why not remove them?

Reply:
yeah, I've removed those now